UnitTestBot/UTBotJava
# Manual testing checklist template correction

## Description

Added checks, added JDK 17

## How to test

### Manual tests

- open UnitTestBot / UTBotJava project
- open Issues tab
- click `New Issue`
- select `Manual testing checklist`
- check text

## Self-check list
